tidb-in-kubernetes/setup.md

```diff
@@ -0,0 +1,155 @@
+# TiDB Operator Setup
```

Comment:
Please add the metadata section. Otherwise, the website build will fail.

Reply:
done